iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
+    
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = NSLocalizedString("slideviewheader", comment: "Text")
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont(name: "Roboto-Regular", size:36.0 )
+
+        labelMMIWInfo.numberOfLines = 0
+        labelMMIWInfo.textColor = UIColor.white
+        labelMMIWInfo.font = UIFont(name: "Roboto-Regular", size:25.0 )
+        labelMMIWInfo.text = NSLocalizedString("slideview1text", comment: "text")
+        
+        pc1View.backgroundColor = UIColor.red
+        pc2View.backgroundColor = UIColor.white
+        pc3View.backgroundColor = UIColor.white
+        pc4View.backgroundColor = UIColor.white
+
+        buttonNext.alpha = 1.0
+        buttonNext.setTitleColor(UIColor.white, for:.normal)
+        buttonNext.setTitleColor(UIColor.green, for:.selected)
+
+        buttonNext.imageEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.contentEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.titleLabel?.font = UIFont.systemFont(ofSize: 17)
+        buttonNext.setTitle("Next", for:.normal)
+        
+
+        buttonNext.addTarget(self, action: #selector(self.changePage(sender:)), for: .touchUpInside)
+        
+        let bottomView = UIView(frame:CGRect(x:0, y:0, width:100, height:50))
+        bottomView.addSubview(pc1View)
+        bottomView.addSubview(pc2View)
+        bottomView.addSubview(pc3View)
+        bottomView.addSubview(pc4View)
+        bottomView.addSubview(buttonNext)
+
+        let gr = UITapGestureRecognizer(target: self, action: #selector(self.changePage(sender:)))
+
+        bottomView.isUserInteractionEnabled = true
+        bottomView.addGestureRecognizer(gr)
+        
+        bottomView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(bottomView)
+        
+        
+        var views: [String:Any] = ["labelMMIW":labelMMIW, "labelMMIWInfo":labelMMIWInfo, "bottomView":bottomView]
+
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-36-[labelMMIW]-56-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views: ["labelMMIW":labelMMIW]))
+ 
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-25-[labelMMIWInfo]-25-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views: ["labelMMIWInfo":labelMMIWInfo]))
+
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-30-[labelMMIW(50)]->=0-[labelMMIWInfo(160)]-12-[bottomView(17)]-36-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views:views))
```

Comment:
Can change this line to make the bottom view (the view with the indicator and Next button) to make clickable area bigger by for example making bottomView(17) be bottomView(37) the height size.  This should be doable by no only increasing the view height, but by making its indicator and Next but. centered.  But there is no way to make a vertical center using visual formate language.  I did try below (see line 134), I think it kind of worked.

Reply:
I might recommend using the standard `someSubview.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true` syntax instead of Visual Format Language. It would make debugging easier and you'd likely be able to solve this.